### ISSUE TITLE
Add support for new object types introduced in Workspace List API

### ISSUE THREAD

**Issue by mohitraj007** [edmunds/databricks-rest-client#99]
Recently the workspace list API seems to be returning new object types (FILE & REPO) (https://docs.databricks.com/dev-tools/api/latest/workspace.html#objecttype) (the type FILE is not listed here yet but we're seeing those in our API responses).
This library has only 3 file types as far as I can see, hence wanted to add support to the new types.
I can make a contribution too but just wanted to report the issue and get the opinion before moving forward.

<img width="777" alt="image" src="https://user-images.githubusercontent.com/14279746/190475664-568bbf7e-c006-4680-a8c6-1a698ae739d9.png">

**Comment by samshuster** [comment]
yes good catch. If you would be willing to contribute this change that would be helpful!
